en/flight_controller/airlink.md

```diff
@@ -0,0 +1,1003 @@
+   - GPU Mali-T864, OpenGL ES1.1/2.0/3.0/3.1
+   - VPU with 4K VP8/9, 4K 10bits H265/H264 60fps Decoding
+   - Remote power control, software reset, power down, RTC Wake-Up, sleep mode
+   - RAM Dual-Channel 4GB LPDDR4
+   - 16GB eMMC
+   - MicroSD up to 256GB
+   - Ethernet 10/100/1000 Native Gigabit
+   - WiFi 802.11a/b/g/n/ac, Bluetooth
+   - USB 3.0 Type C
+   - 2x Video: 4-Lane MIPI CSI (FPV Camera) and 4-Lane MIPI CSI with HMDI Input (Payload Camera)
+
+- **LTE/5G Connectivity Module**
+
+   - Up to 600 Mbps bandwidth
+   - 5G sub-6 and mmWave, SA and NSA operations
+   - 4G Cat 20, up to 7xCA, 256-QAM DL/UL, 2xCA UL
+   - 4 x 4 MIMO for 4G and 5G (sub-6 bands)
+   - 3G HSPA+
+   - Approved by JRL/JTBL, FCC, PTCRB, RED, GCF
+   - Antenna, 4x4 MIMO
+   - Bands: Worldwide
+
+## Set content
+
+@[youtube](https://youtu.be/lex7axW8WQg)
+
+AIRLink set includes everything needed to setup the system and get prepared for the flight. Standard set contains:
+   
+   - 1x AIRLink Enterprise unit
+   - 1x FPV camera with CSI cable
+   - 1x WiFi antenna with MMCX connector
+   - 2x/4x LTE/5G antenna with MMCX connector
+   - 1x HDMI to mini HDMI cable1x set of cables (7 cables for all connectors)
+   
+[AIRLink Telemetry](https://sky-drones.com/sets/airlink-telemetry-set.html) based on the Microhard LAN/IP-based RF micromodule is available as an add-on and is fully compatible with AIRLink.
+
+
+## Editions
+
+AIRLink editions offer different integration levels required by drone manufacturers: Enterprise and Core. AIRLink Enterprise is ideal for a quick start, evaluation and prototyping while Core is optimised for deep integration and mid-high volume manufacturing. 
+
+**AIRLink Enterprise**
+
+![AIRLink Enterprise](../../assets/flight_controller/airlink/airlink-enterprise.jpg)
+
+SmartAP AIRLink's Enterprise edition is intended for prototyping and low to medium volume drone production. Quick and easy installation thanks to the dedicated mounting holes and integrated heatsink for power dissipation.
+
+**AIRLink Core**
+
+SmartAP AIRLink's Core edition is intended for medium to high volume production and deep integration with customer's hardware. It weighs only 89 g and can be attached to a metal frame for optimum cooling.
+
+![AIRLink Core](../../assets/flight_controller/airlink/airlink-core.jpg)
+
+   <table>
+   <tbody>
+   <tr>
+   <th>Parameter </th>
+   <th>AIRLink Enterprise </th>
+   <th>AIRLink Core </th>
+   </tr>
+   <tr>
+   <td>Enclosure</td>
+   <td>Aluminum, with integrated heatsink and fan mounting option.</td>
+   <td>External heatsink or reasonable power dissipation should be provided by the design.</td>
+   </tr>
+   <tr>
+   <td>Dimensions</td>
+   <td>L103 x W61 x H37 mm</td>
+   <td>L100 x W57 x H22 mm</td>
+   </tr>
+   <tr>
+   <td>Weight</td>
+   <td>198 g</td>
+   <td>89 g</td>
+   </tr>
+   <tr>
+   <td>Ambient temperature</td>
+   <td>-40°C-..+50°C</td>
+   <td>-40°C-..+50°C</td>
+   </tr>
+   </tbody>
+   </table>
```

Comment:
Easier to maintain in markdown for this width table. I used https://tableconvert.com/html-to-markdown with simple markdown table setting

```suggestion
Parameter  | AIRLink Enterprise  | AIRLink Core  
--- | --- | ---
Enclosure | Aluminum, with integrated heatsink and fan mounting option. | External heatsink or reasonable power dissipation should be provided by the design. 
Dimensions | L103 x W61 x H37 mm | L100 x W57 x H22 mm 
Weight | 198 g | 89 g 
Ambient temperature | -40°C-..+50°C | -40°C-..+50°C 
```

Reply:
Ditto for other tables.